src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -2026,9 +2026,30 @@ ExtractRangeTableEntryWalker(Node *node, List **rangeTableList)
 	}
 	else if (IsA(node, Query))
 	{
-		walkIsComplete = query_tree_walker((Query *) node,
-										   ExtractRangeTableEntryWalker,
-										   rangeTableList, QTW_EXAMINE_RTES);
+		Query *query = (Query *) node;
+
+		/*
+		 * Since we're only interested in range table entries, we only decend
+		 * into all parts of the query when it is necessary. Otherwise, it is
+		 * sufficient to decent into range table list since its the only part
+		 * of the query that could contain range table entries.
```

Comment:
`decend` and `decent` -> `descend`

Reply:
Two different typos for the same word in the same sentence 😄